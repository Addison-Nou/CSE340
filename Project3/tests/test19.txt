{
    x1 : BOOLEAN;
    x1 = FALSE;
    x2 : BOOLEAN;
    x2 = TRUE;
    WHILE (x1)
        x1 = TRUE;
}