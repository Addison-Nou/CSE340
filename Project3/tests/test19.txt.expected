x1 3 2
x2 5 4
x1 6 2
x1 7 2